fdbserver/SimulatedCluster.actor.cpp

```diff
@@ -758,7 +779,8 @@ void SimulationConfig::generateNormalConfig(int minimumReplication, int minimumR
 		}
 	default:
 		ASSERT(false); // Programmer forgot to adjust cases.
-	}
+	}*/
+	set_config("ssd-rocksdb-experimental"); // not really working yet.  uncomment this line, and commment the above switch to test with RocksDB
```

Comment:
the simulation tests won't pass in CI unless you undo this change:  It's hardcoding test runs to rocksdb.